Servers/swagger.yaml

```
@@ -4336,6 +4338,254 @@ paths:
                   example: internal server error
                 error:
                   type: object
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)